VoodooSMBus/VoodooSMBusControllerDriver.cpp

```diff
@@ -263,8 +269,15 @@ IOWorkLoop* VoodooSMBusControllerDriver::getWorkLoop() {
 
 
 void VoodooSMBusControllerDriver::handleInterrupt(OSObject* owner, IOInterruptEventSource* src, int intCount) {
+    u16 pci_status = 0;
     u8 status;
-
+    
+    // Confirm this is our interrupt
+    pci_status = pci_device->configRead16(SMBPCISTS);
+    if (!(pci_status & SMBPCISTS_INTS)) {
```

Comment:
Why is that necessary? I am just curious...

Reply:
It was from i801.c, not exactly sure it's needed. I noticed it would fire once when it finished initializing but that's the only time I noticed it. We could remove it if we wanted to.